data/sql/updates/pending_db_world/phalanx-location.sql

```diff
@@ -0,0 +1,6 @@
+
+UPDATE creature SET position_x = 848.066, position_y = -230.809, position_z = -43.698, orientation = 1.18512 WHERE guid = 46615;
+
+INSERT INTO waypoint_data (id, point, position_x, position_y, position_z, orientation, delay, move_type, action, action_chance, wpguid) VALUES (32365006, 1, 862.813, -220.542, -43.7036, 1, 0, 2, 0, 100, 0);
+INSERT INTO waypoint_data (id, point, position_x, position_y, position_z, orientation, delay, move_type, action, action_chance, wpguid) VALUES (32365006, 2, 869.566, -226.553, -43.6977, -0.654868, 0, 2, 0, 100, 0);
+INSERT INTO waypoint_data (id, point, position_x, position_y, position_z, orientation, delay, move_type, action, action_chance, wpguid) VALUES (32365006, 3, 867.15, -222.741, -43.6977, 2.18024, 0, 2, 0, 100, 0);
```

Comment:
```
UPDATE creature SET position_x = 848.066, position_y = -230.809, position_z = -43.698, orientation = 1.18512 WHERE guid = 46615;

DELETE FROM `waypoint_data` WHERE `id` = 32365006 AND `point` IN (1, 2, 3);
INSERT INTO waypoint_data (id, point, position_x, position_y, position_z, orientation, delay, move_type, action, action_chance, wpguid) VALUES 
(32365006, 1, 862.813, -220.542, -43.7036, 1, 0, 2, 0, 100, 0),
(32365006, 2, 869.566, -226.553, -43.6977, -0.654868, 0, 2, 0, 100, 0),
(32365006, 3, 867.15, -222.741, -43.6977, 2.18024, 0, 2, 0, 100, 0);
```

Reply:
thats a reformat of the sql to mean [Azerothcore's SQL Standards ](https://www.azerothcore.org/wiki/sql-standards).